build.sbt

```diff
@@ -120,6 +120,9 @@ lazy val sparkEtl = Project(id = "spark-etl", base = file("spark-etl")).
   dependsOn(spark, s3, accumulo).
   settings(commonSettings: _*)
 
+lazy val index = Project("index", file("index")).
+  settings(commonSettings: _*)
```

Comment:
I've got no clue. When writing the docs up, I actually saw an index folder at root and deleted it. I didn't anticipate it also showing up in the build file